website/docs/foundations/border/partials/border-radius.md

```diff
@@ -0,0 +1,12 @@
+## Border radius
+
+Figma doesn't currently support border radius as a style and the HashiCorp Design System doesn't yet publish tokens for border radius. These are a set of high-level guidelines for how border-radius is used in components of various types.
+
+At this time the strategy for applying border radius is determined by the size of the component.
```

Comment:
It could be more concise and use an active voice. I'm torn on whether "border radius" should be "border-radius," and Grammarly can't seem to make up its mind, so I left it be. 🤷🏻‍♀️ 

```suggestion
We don't currently offer border radius tokens or Figma styles.

The size of the component determines the border radius. These are high-level guidelines for how border radius is used in our components.
```